tests/integration-tests/cypress/fixtures/config.json

```diff
@@ -4,6 +4,6 @@
   "adminUsername":"admin_username",
   "adminPass":"admin_password",
   "CC_TERMINAL_NAME":"Your CC Terminal Name",
-  "KLARNA_DKK_TERMINAL_NAME":"Your Klarna Terminal Name"
-}
-
+  "KLARNA_DKK_TERMINAL_NAME":"Your Klarna Terminal Name",
+  "iDEAL_EUR_TERMINAL": "Your iDEAL Terminal Namel"
```

Comment:
@saadidrees711 please fix the typo to **Your iDEAL Terminal Name**

Reply:
@maf-embraceit typo has been fixed.